tla-bmcmt/src/main/scala/at/forsyte/apalache/tla/bmcmt/rewriter2/Rule.scala

```diff
@@ -0,0 +1,14 @@
+package at.forsyte.apalache.tla.bmcmt.rewriter2
+
+import at.forsyte.apalache.tla.lir.TlaEx
+
+/**
+ * A rule in the new rewriting system. It can be tested, or applied. In the latter case, it either returns
+ * a TlaEx (rewritten expression) or throws a RewritingException (represented by computation)
```

Comment:
I don't understand this part: `throws a RewritingException (represented by computation)`.